readthedocsext/theme/templates/organizations/partials/header.html

```diff
@@ -20,6 +20,16 @@
 {% endcomment %}
 
 {% block organization_header %}
+  {# Render all the notifications attached to the organization.#}
+  {% for notification in organization.notifications.all %}
+    <div class="ui message">
+      <i class="fa-solid {{ notification.get_message.get_display_icon_classes }}"></i>
```

Comment:
So, two changes required here:

```suggestion
      <i class="{{ notification.get_message.get_display_icon_classes }} icon"></i>
```

Drop the `fa-solid` class that is already defined in the icon class method, and `icon` is a required class here, from FUI.